### ISSUE TITLE
[docs-v2.2.0]: Fix the broken URL in slack code of conduct

### ISSUE THREAD

**Issue by withshubh** [ToolJet/ToolJet#6401]
### Summary

The hyperlink of "The ADA Initiative" in the [Enforcement](https://docs.tooljet.com/docs/contributing-guide/slackcoc/#enforcement) section is broken.

Broken URL: http://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/

Solution:
- Go to the `docs/versioned_docs/version-2.2.0/contributing-guide/slackcoc.md`
- On line number 63 change the URL to `http://adainitiative.org/`

### Issue Type

Documentation bug

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

**Comment by Techylawyer** [comment]
Kindly assign this issue to me

**Comment by tejakummarikuntla** [comment]
Thank you for your interest, @Techylawyer, Assigned this issue to you.

**Comment by withshubh** [comment]
After careful consideration, we have reached a decision to consolidate all the issues that necessitate changes in each version into a single merged issue. This resolution has been implemented with the aim of minimizing the inconvenience associated with creating separate issues or pull requests for identical bugs, as an affect this issues needs to be closed.

We appreciate everyone's efforts in identifying these issues and submitting them for consideration. We hope you understand and are sorry for the inconvenience caused, would encourage you to explore other `good first issues` and `up for grabs`.
